src/geckomat/change_model/makeEcModel.m

```diff
@@ -103,6 +103,16 @@
     ec.geckoLight=false;
 end
 
+%Check for conflicting reaction and metabolite identifiers
+conflictId = startsWith(model.mets,'prot_');
```

Comment:
This is validation of the traditional GEM, right ? Because ecModel uses `prot_` or `draw_`

Reply:
Exactly, `model` at this point is the traditional GEM that is used as input. Already having metabolites and reactions with these suffixes would undoubtly cause problems later on (which might not be so easy to trace back), so best to avoid them (and it probably doesn't affect many models).